pkg/virt-controller/services/template_test.go

```diff
@@ -1036,7 +1037,7 @@ var _ = Describe("Template", func() {
 
 				pod, err := svc.RenderLaunchManifest(&vmi)
 				Expect(err).ToNot(HaveOccurred())
-				value, ok := pod.Annotations["v1.multus-cni.io/default-network"]
+				value, ok := pod.Annotations[multus.DefaultNetworkCNIAnnotation]
```

Comment:
I would leave the the annotation hard-coded in tests to make it fail in case the definition is change, making such change "not go silently", make sense?

Reply:
I see your point, but please note that when using an IDE you cannot see where `DefaultNetworkCNIAnnotation` is tested in case its not used as a constant.
One can search the string literal, but I personally find it annoying.

Reply:
In general I wouldn't change test because its not continent to find where it is tested if the mentioned principal should be applied (where unit-test hardcode the mentioned const) 

Following offline discussion, it was said that this is an intermediate state before doing additional code movement, once completed, test should reverted to the original form (having the const hard-coded).